tsjs.js

```diff
@@ -26,7 +27,14 @@ process.on('exit', () => {
 
 try {
     fs.writeFileSync(tempTsfmtFile, JSON.stringify(require(tsfmtPath)));
-    fs.writeFileSync(tempTsLintFile, JSON.stringify(require(tslintPath)));
+    fs.writeFileSync(
+        tempTsLintFile,
+        flow(
+            update('no-unused-variable[1].ignore-pattern', (val) => argv.ignorepattern || val),
+            JSON.stringify,
+        )(require(tslintPath)),

```

Comment:
`flow` acts pretty much the same as redux `compose` . see prior lodash for difference, https://github.com/coveo/tsjs/pull/11/commits/502a6b2848911fa887d9e99a0723615677a0dbd0  

seems much cleaner like this to me, on `flow` https://medium.com/making-internets/why-using-chain-is-a-mistake-9bc1f80d51ba